eng/cake/dotnet.cake

```diff
@@ -150,117 +150,6 @@ Task("dotnet-samples")
         }, binlogPrefix: "sample-");
     });
 
-Task("dotnet-templates")
-    .Does(() =>
-    {
-        if (localDotnet)
-            SetDotNetEnvironmentVariables();
-
-        var dn = localDotnet ? dotnetPath : "dotnet";
-
-        var tempDir = PrepareSeparateBuildContext(
-            "templatesTest",
-            props: "./src/Templates/tests/Directory.Build.props",
-            targets: "./src/Templates/tests/Directory.Build.targets");
```

Comment:
This feature may also need to be added? I use this to add some errors if I detect an incorrect build. Right now it is just on Windows to make sure that it is using the correct build tools otherwise machines that do not have VS will not build correctly.